### PR TITLE
6 get comments by article

### PR DESCRIPTION
**Pull request by Veresio**

### CODE REVIEW

File: __tests__/server.test.js [hunk 1]
```diff
@@ -111,3 +111,61 @@ describe("GET /api/articles/:article_id", () => {
...
+  });
+
+  test("should return a list of all the comments attached to that article id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(11);
+        body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
```
Comment (two-triangles) [inline __tests__/server.test.js:128]: Please add the following to your test: 

 - check that the typeof the property value is what you expect as well as checking that is has the property keys you expect.

File: __tests__/server.test.js [hunk 2]
```diff
@@ -111,3 +111,61 @@ describe("GET /api/articles/:article_id", () => {
...
+  test('should return a 404 error with the message "Article not found" when provided an incorrect id', () => {
+    return request(app)
+      .get("/api/articles/70650/comments")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("Article not found");
+      });
+  });
+
+  xtest("should return a 404 with 'No comments available' if there are no comments attached to the article", () => {
```
Comment (two-triangles) [inline __tests__/server.test.js]: This test won't pass currently. Remove the x and implement the behaviour needed to make it pass 👍 !

File: __tests__/server.test.js [hunk 3]
```diff
@@ -152,7 +159,7 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 
-  xtest("should return a 404 with 'No comments available' if there are no comments attached to the article", () => {
+  test("should return a 404 with 'No comments available' if there are no comments attached to the article", () => {
```
Comment (Beth-McCormack) [inline __tests__/server.test.js:162]: If there are no comments, is this an error? The request happened with no error, so would be a 200 and return an empty array

**Review by Beth-McCormack — COMMENTED** [review]
I've left a comment, feel free to merge once done

File: __tests__/server.test.js (hunk 3 above)
Comment (Veresio) [reply __tests__/server.test.js:162]: Very good point, I didn't even consider that, thanks

**Pull request merged**